### PR TITLE
Fixes a bug identified by @JenkeScheen in #236

### PR DESCRIPTION
**Pull request by dotsdl**
`status` was a module import, but was being overriden by the use of `status` as an arg in certain API endpoints.

This PR fixes this issue.

### CODE REVIEW

**Comment by dotsdl** [comment]
Can I get a review on this one @ianmkenney?

File: alchemiscale/compute/api.py [hunk 1]
```diff
@@ -9,7 +9,8 @@
 import json
 from datetime import datetime, timedelta
 
-from fastapi import FastAPI, APIRouter, Body, Depends, HTTPException, status
+from fastapi import FastAPI, APIRouter, Body, Depends, HTTPException
+from fastapi import status as http_status
```
Comment (ianmkenney) [inline alchemiscale/compute/api.py:13]: Currently, `compute.api` doesn't use `HTTPException` or `status`. We can just remove them.

Comment (ianmkenney) [inline alchemiscale/compute/api.py:13]:
```suggestion
from fastapi import FastAPI, APIRouter, Body, Depends
```

**Comment by ianmkenney** [comment]
Other than the unused imports, seems sensible to me

File: alchemiscale/compute/api.py (hunk 1 above)
Comment (dotsdl) [reply alchemiscale/compute/api.py:13]: We should start using them in various places to give more descriptive error messages, so I'd rather leave them in as a kind of placeholder.

Comment (dotsdl) [reply alchemiscale/compute/api.py]: Accepted your suggestion; can proceed later with adding these back in as needed.

**Pull request merged**